cmd/krel/cmd/root.go

```diff
@@ -68,6 +69,8 @@ func init() {
 	rootCmd.PersistentFlags().BoolVar(&rootOpts.cleanup, "cleanup", false, "cleanup flag")
 	rootCmd.PersistentFlags().StringVar(&rootOpts.repoPath, "repo", filepath.Join(os.TempDir(), "k8s"), "the local path to the repository to be used")
 	rootCmd.PersistentFlags().StringVar(&rootOpts.logLevel, "log-level", "info", "the logging verbosity, either 'panic', 'fatal', 'error', 'warn', 'warning', 'info', 'debug' or 'trace'")
+
+	rootCmd.AddCommand(anago.AnagoCmd)
```

Comment:
Init functions are triggered "the other way around" which means that we now have to explicitly add the subcommand there.